themes/icore-open/color-schemes/open-color.scss

```diff
@@ -0,0 +1,399 @@
+//
+//
+//  𝗖 𝗢 𝗟 𝗢 𝗥
+//  v 1.6.3
+//
+//  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
```

Comment:
Last version is v1.9.1
Maybe also add a reference / link to GitHub here?

https://github.com/yeun/open-color/blob/master/open-color.scss